po/pt_BR.po

```diff
@@ -489,14 +508,16 @@ msgstr "Criptografia"
 
 #: ../src/extract_dialog.c
 msgid "Ensure a containing directory"
-msgstr "Garantir um diretório de extração"
+msgstr "Certifique-se da existência de um diretório de destino\npara a extração "
```

Comment:
Why does the translation of the checkbox label have to be as long as the entire tooltip help text?